lib/properties.ts

```diff
@@ -171,9 +173,9 @@ export class CompilerProps {
      * @returns {*} Transformed value(s) found or fn(defaultValue)
      */
     get(
-        langs: string | LanguageDef[],
+        langs: string | LanguageDef[] | Record<string, any>,
```

Comment:
It appears compiler-finder can invoke .get with a `Record<string, any>` from https://github.com/compiler-explorer/compiler-explorer/blob/main/lib/languages.ts#L588

Reply:
Got it - maybe we should try and fix that (putting a note there for now, not as this PR? just so we don't forget in either place?)

Reply:
Yeah, just noting for now. Evidently the type does still work, but something that might be worth revisiting.